server/src/main/java/io/spine/server/tenant/IdInTenant.java

```diff
@@ -88,13 +89,21 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) {
             return false;
         }
-        IdInTenant<?> tenant = (IdInTenant<?>) o;
-        return Objects.equals(id, tenant.id) &&
-                Objects.equals(tenantId, tenant.tenantId);
+        IdInTenant<?> another = (IdInTenant<?>) o;
+        return Objects.equals(id, another.id) &&
+                Objects.equals(tenantId, another.tenantId);
     }
 
     @Override
     public int hashCode() {
         return Objects.hash(id, tenantId);
     }
+
+    @Override
+    public String toString() {
```

Comment:
Please use `ToStringHelper` in this case and in similar cases in the change set.